internal/host/plugin/ids.go

```diff
@@ -10,16 +10,19 @@ import (
 )
 
 func init() {
-	if err := subtypes.Register(host.Domain, Subtype, HostCatalogPrefix, HostSetPrefix, HostPrefix); err != nil {
+	if err := subtypes.Register(host.Domain, Subtype, HostCatalogPrefix, PreviousHostCatalogPrefix, HostSetPrefix, PreviousHostSetPrefix, HostPrefix, PreviousHostPrefix); err != nil {
 		panic(err)
 	}
 }
 
 // PublicId prefixes for the resources in the plugin package.
 const (
-	HostCatalogPrefix = "hc"
-	HostSetPrefix     = "hs"
-	HostPrefix        = "h"
+	HostCatalogPrefix         = "hcplg"
+	PreviousHostCatalogPrefix = "hc"
+	HostSetPrefix             = "hs"
+	PreviousHostSetPrefix     = "hsplg"
+	HostPrefix                = "h"
+	PreviousHostPrefix        = "hplg"
```

Comment:
The choice of naming here seems inconsistent, do we really want the _new_ `HostCatalog` prefix to include the `plg` suffix when the other new ones don't?

Reply:
Oh weird

How did that happen